ForgeRock Cloud APIs.postman_collection.json

```diff
@@ -4258,6 +4269,526 @@
 						}
 					],
 					"_postman_isSubFolder": true
+				},
+				{
+					"name": "Email",
+					"item": [
+						{
+							"name": "Get Email Templates",
+							"request": {
+								"auth": {
+									"type": "bearer",
+									"bearer": [
+										{
+											"key": "token",
+											"value": "{{accessToken}}",
+											"type": "string"
+										}
+									]
+								},
+								"method": "GET",
+								"header": [
+									{
+										"warning": "This is a duplicate header and will be overridden by the Authorization header generated by Postman.",
+										"key": "Authorization",
+										"value": "Bearer {{accessToken}}",
+										"description": "Requires the `access_token` for the hosting environment."
+									},
+									{
+										"key": "Content-Type",
+										"name": "Content-Type",
+										"type": "text",
+										"value": "application/json"
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": ""
+								},
+								"url": {
+									"raw": "{{tenantApiV1Url}}/email-templates",
+									"host": [
+										"{{tenantApiV1Url}}"
+									],
+									"path": [
+										"email-templates"
+									]
+								},
+								"description": "Returns a list of available email templates, in JSON format, classified by `name`. For more information, see our documentation on [Email Templates](https://developer-cloud.forgerock.com/quickstarts/email-templates/).\n\nTo send a test email via REST, review the output for the desired template `type` or template `id`. You can use that information in the POST Send Test Email REST call."
```

Comment:
I'm just wondering if we can make the description a bit friendlier.  Gets a list of the available email templates.  This might just be part of a larger discussion on how we can just sound a bit more conversational in the API docs and if that is appropriate here.

Reply:
Here's my first attempt (and I've tried to make other descriptions in this PR more friendly as well)
![Screenshot from 2019-04-03 08-05-34](https://user-images.githubusercontent.com/3287976/55489729-5a29b580-55e7-11e9-807e-03986f8bf310.png)